src/pages/Bookings.tsx

```diff
@@ -103,55 +84,51 @@ export const Bookings = () => {
       </MessageBox>
 
       <MessageBox type="warn" show={isAccountContract}>
-        It seems that your connected account is a multisig wallet or other smart contract. It is not possible to authorize a smart contract address.
+        It seems that your connected account is a multisig wallet or other smart contract.
+        It is not possible to authorize a smart contract address.
       </MessageBox>
 
       <MessageBox type="error" show={!!error}>
         <Box>{error}</Box>
       </MessageBox>
 
-      {isLoggedIn &&
+      {isLoggedIn && (
         <Box margin={theme.spacing(5)}>
-          <Button
-            variant="contained"
-            onClick={logout}
-          >
+          <Button variant="contained" onClick={logout}>
             LogOut
           </Button>
         </Box>
-      }
+      )}
 
-      {bookings.length > 0 &&
+      {bookings.length > 0 && (
         <>
           <Box
             sx={{
               display: 'flex',
               alignItems: 'center',
-              justifyContent: 'space-between',
+              justifyContent: 'space-between'
             }}
           >
             <Box padding={theme.spacing(2)}>N</Box>
             <Box padding={theme.spacing(2)}>offerId</Box>
             <Box padding={theme.spacing(2)}>orderId</Box>
           </Box>
-          {bookings.map(
-            (booking, index) => (
-              <Box
-                key={index}
-                sx={{
-                  display: 'flex',
-                  alignItems: 'center',
-                  justifyContent: 'space-between',
-                }}
-              >
-                <Box padding={theme.spacing(2)}>{index}</Box>
-                <Box padding={theme.spacing(2)}>{booking.offerId}</Box>
-                <Box padding={theme.spacing(2)}>{booking.orderId}</Box>
-              </Box>
-            )
-          )}
+          {bookings.map((booking, index) => (
+            <Box
```

Comment:
There is also a `Stack` component, which is a Box/Div with build in functionality for displaying items using `flex`.  no need to change, just sharing knowledge.